src/Nethermind/Nethermind.AuRa/Validators/ContractValidator.cs

```diff
@@ -51,9 +52,10 @@ public class ContractValidator : AuRaValidatorProcessorBase
         
         private ValidatorContract _validatorContract;
         private PendingValidators _currentPendingValidators;
-        private long _lastProcessedBlockNumber = -1;
+        private long _lastProcessedBlockNumber = 0;
```

Comment:
unnecessary init (super minor, ignore :)

Reply:
Yes, but I want to explicitly set that default as its not accidental that it changed from -1 to 0 :)